recipes_source/recipes_index.rst

```diff
@@ -278,7 +278,7 @@ Recipes are bite-sized, actionable examples of how to use specific PyTorch featu
    :header: CPU launcher script for optimal performance on Intel® Xeon
```

Comment:
One thing I noticed that the in the card you call the tutorial "CPU launcher script for optimal performance on Intel® Xeon" and in the actual [page](https://docs-preview.pytorch.org/pytorch/tutorials/2989/recipes/xeon_run_cpu.html) it's "Optimizing PyTorch Inference with Intel® Xeon® Scalable Processors" - you might want to keep the same name to avoid the confusion. 
<img width="1213" alt="Screenshot 2024-08-01 at 10 53 13 AM" src="https://github.com/user-attachments/assets/231d2879-2ee9-4f92-9058-404d4ddb8ca6">

Reply:
Thanks for finding this. The title is aligned as "Optimizing CPU Performance on Intel® Xeon® with run_cpu Script"